networking_calico/datamodel_v3.py

```diff
@@ -64,8 +64,12 @@ def put(resource_kind, namespace, name, spec, annotations={}, labels=None,
       unchanged, and existing annotations with the same keys are overwritten by
       these new values.
 
-    - mod_revision (string): If specified, indicates that the write should only
-      proceed if replacing an existing value with that mod_revision.
+    - mod_revision: If 0 (a number), indicates that the write should only
```

Comment:
Is the `0` case used anywhere?  Is didn't see it.

Reply:
There's an example here: https://github.com/projectcalico/networking-calico/blob/master/networking_calico/plugins/ml2/drivers/calico/endpoints.py#L102